src/cli/README.md

```diff
@@ -1169,6 +1169,25 @@ DNS response for example.com. - fd4c:9574:3720:2:0:0:5db8:d822 TTL:20456
 Done
 ```
 
+### dns resolve4 \<hostname\> \[DNS server IP\] \[DNS server port\] \[response timeout (ms)\] \[max tx attempts\] \[recursion desired (boolean)\]
+
+Send DNS query to obtain IPv4 address for a given hostname.
+
+Requires `OPENTHREAD_CONFIG_DNS_CLIENT_NAT64_ENABLE`.
+
+The parameters after `hostname` are optional. Any unspecified (or zero) value for these optional parameters is replaced by the value from the current default config (`dns config`).
+
+This command requires a NAT64 prefix to be configured and present in Thread Network Data.
+
+The output provides the NAT64 synthesized IPv6 addresses for the IPv4 addresses from the query response.
+
+Example below is with NAT64 prefix `2001:db8:122:344::/96` being used within Thread mesh. The outputted IPv6 address corresponds to `142.250.191.78` IPv4 address.
```

Comment:
For example, if a NAT64 prefix of `2001:db8:122:344::/96` is used within the Thread mesh, the outputted IPv6 address corresponds to an IPv4 address of `142.250.191.78` for the `ipv4.google.com` host:

Reply:
Changed in new push.